package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "start": "pm2 start .pm2.json",
     "start:debug": "SILEX_DEBUG=true node dist/server/server.js",
-    "build": "npm run build:css && npm run build:html && npm run build:js:release ",
```

Comment:
i don't know much more about the Silex architecture, i thought it just build the final release, is that right?After remove the java build phase, i deployed the site successfully, maybe some functionality not works well, is there any function works using the `admin.js` ? For now, the simple usage works fine, here the [site](http://yadesigner.au-syd.mybluemix.net/) i deployed, would mind verify that